buildconfig/config_emsdk.py

```diff
@@ -0,0 +1,239 @@
+"""Config on Unix"""
```

Comment:
probably better to expand on this docstring with better related documentation

Reply:
-on +like